pkg/controller/kogitoapp/resource/build_config_s2i.go

```diff
@@ -36,6 +36,9 @@ const (
 	buildS2IlimitCPUEnvVarKey    = "LIMIT_CPU"
 	buildS2IlimitMemoryEnvVarKey = "LIMIT_MEMORY"
 	mavenMirrorURLEnvVar         = "MAVEN_MIRROR_URL"
+	projectGroupId               = "PROJECT_GROUP_ID"
+	projectArtifactId            = "PROJECT_ARTIFACT_ID"
+	projectVersion               = "PROJECT_VERSION"
```

Comment:
Please end the var names with `EnvVar` suffix

Reply:
I would suggest to use mavenGroupIdEnvVar and so on.

Reply:
changed name to `mavenGroupIdEnvVar`